### DISCUSSION TITLE
Does Cardinal recognize default system UI scale?

### DISCUSSION THREAD

**Discussion by swartzfeger** [DISTRHO/Cardinal#356] [General]
Ubuntu 22.04 -- does Cardinal recognize default system scale?

I'm switching from macOS to Linux, so I installed Ubuntu on my Macbook Pro (retina/hi dpi screen). Due to the display resolution, I have my default system scale set to 200%, otherwise everything is too small.

When I instantiated Cardinal for the first time, it looked like it had defaulted to a 100% UI scale. I managed to go into settings and zoom to 200% which made the rack readable, but the menus were still too small.

Is there a way to fix this? Is it even 'broken'? Thank you!

**Comment by falkTX** [comment]
Cardinal uses DPF which has ways to check for this.
If you use it in a DAW, it is up to that DAW to give the correct scaling to Cardinal as plugin.

For standalone, Cardinal will read the value from X11. It works in KDE which I used to test daily.
You can force the scaling of the standalone by exporting an env var before running it, like so:

```
export DPF_SCALE_FACTOR=2.0
Cardinal
```

**Reply by swartzfeger** [reply]
That makes sense, thank you Felipe!
